code/modules/projectiles/projectile/bullets.dm

```diff
@@ -275,6 +275,26 @@
 	icon = 'icons/obj/chemical.dmi'
 	icon_state = "syringeproj"
 
+/obj/item/projectile/bullet/dart/syringe/pierce_ignore
+	piercing = TRUE
+
+/obj/item/projectile/bullet/dart/syringe/pierce_ignore/on_hit(atom/target, blocked = 0, hit_zone)
+	if(iscarbon(target))
+		var/mob/living/carbon/M = target
+		if(blocked != INFINITY)
+			if(!(istype(M.wear_suit, /obj/item/clothing/suit/space/deathsquad) || istype(M.wear_suit, /obj/item/clothing/suit/space/hardsuit/syndi/elite/sst)))
```

Comment:
Do you think you could add a trait or something to these suits rather than snowflaking this?

Reply:
It has a new flag now, I wasn't really happy with the `istype()` too ngl